src/partials/welcome.html

```diff
@@ -205,4 +205,10 @@ <h2 class="instructions" translate>welcome.ALREADY_CONNECTED</h2>
         </div>
 
     </div>
+
+    <a ng-if="!ctrl.showLoadingIndicator && ctrl.lastConnectedClientOS === 'ios'" class="desktop-beta-button" href="https://threema.ch/download-md">
+        <span class="badge">Beta</span>
+        <span translate>welcome.DESKTOP_BETA_HINT</span>
+        <md-icon class="material-icons md-24">arrow_forward</md-icon>
+    </a>
```

Comment:
If you set a password, then the badge is offset from the bottom:

![screenshot-20240611-181744](https://github.com/threema-ch/threema-web/assets/78751145/9402b120-8b20-45f6-894b-dcc449dcdc68)

That's probably not intended, right?

Reply:
No, that's definitely incorrect. Thanks! Amended.